packages/cmf/src/bootstrap.js

```diff
@@ -0,0 +1,136 @@
+
+export const bactchedSubscribe = batchedSubscribe(notify => {
+	requestAnimationFrame(notify);
+});
+
+export function bootstrapRegistry(options) {
+	assertTypeOf(options, 'sagas', 'object');
+	assertTypeOf(options, 'components', 'object');
+	assertTypeOf(options, 'expressions', 'object');
+	assertTypeOf(options, 'actionCreators', 'object');
+	registerInternals();
+	if (options.components) {
+		component.registerMany(options.components);
+	}
+	if (options.expressions) {
+		expression.registerMany(options.expressions);
+	}
+	if (options.actionCreators) {
+		actionCreator.registerMany(options.actionCreators);
+	}
+	if (options.sagas) {
+		sagas.registerMany(options.sagas);
+	}
+}
+
+export function bootstrapSaga(options) {
+	assertTypeOf(options, 'saga', 'function');
+	function* cmfSaga() {
+		yield fork(sagas.component.handle);
+		if (options.sagaRouterConfig) {
+			// eslint-disable-next-line no-console
+			console.warn("sagaRouter is deprecated please use cmfConnect 'saga' props");
+			yield fork(sagaRouter, options.history || hashHistory, options.sagaRouterConfig);
+		}
+		if (typeof options.saga === 'function') {
+			yield call(options.saga);
+		}
+	}
+	const middleware = createSagaMiddleware();
+	return {
+		middleware,
+		run: () => middleware.run(cmfSaga),
+	};
+}
+
+export function bootstrapRedux(options, sagaMiddleware) {
+	assertTypeOf(options, 'settingsURL', 'string');
+	assertTypeOf(options, 'preReducer', 'function');
+	assertTypeOf(options, 'httpMiddleware', 'function');
+	assertTypeOf(options, 'enhancer', 'function');
+	assertTypeOf(options, 'preloadedState', 'object');
+	assertTypeOf(options, 'middlewares', 'Array');
+	assertTypeOf(options, 'storeCallback', 'function');
+	assertTypeOf(options, 'reducer', ['object', 'function']);
+
+	if (options.preReducer) {
+		storeAPI.addPreReducer(options.preReducer);
+	}
+	if (typeof options.httpMiddleware === 'function') {
+		storeAPI.setHttpMiddleware(options.httpMiddleware);
+	}
+	let enhancer = bactchedSubscribe;
+	if (typeof options.enhancer === 'function') {
+		enhancer = compose(options.enhancer, bactchedSubscribe);
+	}
+	const middlewares = options.middlewares || [];
+	const store = storeAPI.initialize(options.reducer, options.preloadedState, enhancer, [
+		...middlewares,
+		sagaMiddleware,
+	]);
+	if (options.settingsURL) {
+		store.dispatch(actions.settings.fetchSettings(options.settingsURL));
+	}
+	if (typeof options.storeCallback === 'function') {
+		options.storeCallback(store);
+	}
+	return store;
+}
+
+/**
+ * This is the function to use in your app index.js file.
+ * It take your configuration and provide a very good default one.
+ * By default it start react with the following addons:
+ * - react-router
+ * - redux
+ * - redux-saga
+ * @param {object} options the set of supported options
+ * @returns {object} app object with render function
+ */
+export default function bootstrap(unSafeOptions) {
+	const options = unSafeOptions || {};
```

Comment:
Use default value please
`export default function bootstrap(options = {}) {`

And this will match the jsDoc param name